doc/pages/Manual.md

```diff
@@ -494,7 +494,9 @@ the package being defined.
   whether the package is installed
 - <a id="pkgvar-enable">`enable`</a>:
   takes the value "enable" or "disable" depending on whether the package is
-  installed
```

Comment:
A key point that was in the original change that's now removed here is that `enable` is not a variable. Since it's listed under the heading "Package variables", I think it needs to be clarified that it is not a variable. The original reason I opened this PR is I mistakenly tried to use it as a variable which led to confusion.

Reply:
Done. What do you think?

Reply:
Looks good, thanks!